pkg/snmp/traps/validation.go

```diff
@@ -24,5 +24,5 @@ func validateCredentials(p *gosnmp.SnmpPacket, c *Config) error {
 		}
 	}
 
-	return errors.New("Unknown community string")
+	return errors.New("unknown community string")
```

Comment:
Since we know handle v2 and v3, I think we might need a more generic message since it's not a community string when using v3.

Reply:
Sounds good 👍

Reply:
Not an issue with this PR, but shall we have the community string part of the error ?


```
	return errors.New("unknown community string: " + p.Community)
```

Reply:
No, the community string is supposed to be a private parameter that we should hide from logs and flares

Reply:
My bad 😮‍💨 , of course !!